src/Data/MongoDBMock/doggos.js

```diff
@@ -0,0 +1,70 @@
+const doggos = [
+  {
+    dogId: 'VIII-27165',
+    dogName: 'Nosek',
+    kennelName: 'Narnia Baltica',
+    pkr: 'VIII-27165',
+    registrationNumber: '5491/XLII/11',
+    breed: 'Golden Retriever',
+    dateOfBirth: new Date(2011, 4, 22),
+    colour: 'gold',
+    chipOrTatoo: 616093900044541,
+    owner: 'Jan Kowalski',
```

Comment:
I;m not sure, can we here use just `ownerId`  with id and not `owner` with name?

Reply:
owner is not a participant, owner lives just here, in that place and is needed only to put on result card. 
It is not important person here (mostly owner is participant, but I think this data are because sometimes owner is someone else, and for documentation organization needs to have it)